composer.json

```diff
@@ -10,9 +10,11 @@
   ],
   "require": {
     "php": ">=7.1",
+    "spryker/quote": "^2.0.0",
```

Comment:
@SpiderDan98 did you consider all possible versions?

Reply:
@daniel-rose No, but everything's okay here.